Cargo.toml

```diff
@@ -2,8 +2,20 @@
 name = "tmsu"
 version = "0.1.0"
 authors = ["Paul Ruane <paul.ruane@oniony.com>"]
+edition = "2018"
 
-[dependencies.clap]
-version = "~2.27" 
-default-features = false
-features = [ "suggestions", "wrap_help" ]
+# TODO: try to use fewer features of dependencies when possible
+[dependencies]
+# CLI dependencies
+structopt = "0.3.14"
+clap = "2.33.1"
```

Comment:
Except one single direct use of `clap` (see comment below), `structopt` is used to define CLI behavior in a clear, data-driven way.